defaults/main.yml

```diff
@@ -4,6 +4,7 @@ bigdft_src: bigdft-suite-{{ bigdft_version }}
 bigdft_src_archive: "{{ bigdft_src }}.tar.gz"
 bigdft_code_folder: "/tmp"
 bigdft_prefix: "/usr/local"
+bigdft_python_path: "{{ ansible_python_interpreter | default('/usr/bin/python') }}"
```

Comment:
so... if `ansible_python_interpreter` is explicitly set to python2 this would fail?

Reply:
Why do you say that? I don't think so

Reply:
Since you seem to rely on the fact that python3 is being used (?)

Reply:
No I install both python(2)-dev/pip and python3-dev/pip; precisely because we can't be sure whether the interpreter will be python 2 or 3. This still works for python2

Reply:
Ok, then I suggest we just proceed and open an issue for this to keep it in mind

Reply:
👍